modules/kubernetes-addons/kyverno/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "helm_config" {
```

Comment:
You are only using one input for all three helm charts.

You need a dedicated helm config variable for each helm chart

Reply:
Thanks. Fixed in the recent commit.